flowbased-computation/flowbased-computation-api/src/main/java/com/farao_community/farao/flowbased_computation/FlowbasedComputation.java

```diff
@@ -0,0 +1,197 @@
+     */
+    public static Runner find(String name) {
+        return find(name, PROVIDERS_SUPPLIERS.get(), PlatformConfig.defaultConfig());
+    }
+
+    /**
+     * Get a runner for default flowbased implementation.
+     *
+     * @throws FaraoException in case we cannot find a default implementation
+     * @return a runner for default flowbased implementation
+     */
+    public static Runner find() {
+        return find(null);
+    }
+
+    /**
+     * A variant of {@link FlowbasedComputation#find(String)} intended to be used for unit testing that allow passing
+     * an explicit provider list instead of relying on service loader and an explicit {@link PlatformConfig}
+     * instead of global one.
+     *
+     * @param name name of the flowbased implementation, null if we want to use default one
+     * @param providers flowbased provider list
+     * @param platformConfig platform config to look for default flowbased implementation name
+     * @return a runner for flowbased implementation named {@code name}
+     */
+    public static Runner find(String name, List<FlowbasedComputationProvider> providers, PlatformConfig platformConfig) {
+        Objects.requireNonNull(providers);
+        Objects.requireNonNull(platformConfig);
+
+        if (providers.isEmpty()) {
+            throw new FaraoException("No flowbased providers found");
+        }
+
+        // if no flowbased implementation name is provided through the API we look for information
+        // in platform configuration
+        String flowbasedName = name != null ? name : platformConfig.getOptionalModuleConfig("flowbased-computation")
+                .flatMap(mc -> mc.getOptionalStringProperty("default"))
+                .orElse(null);
+        FlowbasedComputationProvider provider;
+        if (providers.size() == 1 && flowbasedName == null) {
+            // no information to select the implementation but only one provider, so we can use it by default
+            // (that is be the most common use case)
+            provider = providers.get(0);
+        } else {
+            if (providers.size() > 1 && flowbasedName == null) {
+                // several providers and no information to select which one to choose, we can only throw
+                // an exception
+                List<String> flowbasedNames = providers.stream().map(FlowbasedComputationProvider::getName).collect(Collectors.toList());
+                throw new FaraoException("Several flowbased implementations found (" + flowbasedNames
+                        + "), you must add configuration to select the implementation");
+            }
+            provider = providers.stream()
+                    .filter(p -> p.getName().equals(flowbasedName))
+                    .findFirst()
+                    .orElseThrow(() -> new FaraoException("FlowBased computation provider '" + flowbasedName + "' not found"));
+        }
+
+        return new Runner(provider);
+    }
```

Comment:
Now you can use PowSyBl PlatformConfigNamedProvider.Finder methods to perform this